tests/integration/test_submissions.py

```diff
@@ -322,6 +322,25 @@ async def test_crud_submissions_works(self, client_logged_in, project_id):
             assert res["drafts"] == [], "there are drafts in submission, expected empty"
             assert len(res["metadataObjects"]) == 4, "submission metadataObjects content mismatch"
 
+        # check that submission cannot be updated and objects cannot be added into published submission
```

Comment:
Objects should not be able to be updated or deleted, either. Is it already tested?

Reply:
- delete should be checked here - https://github.com/CSCfi/metadata-submitter/blob/develop/metadata_backend/api/handlers/object.py#L246
- updated here: https://github.com/CSCfi/metadata-submitter/blob/develop/metadata_backend/api/handlers/object.py#L332

though not sure we have integration or unit tests for these

Reply:
These should all be tested now.